explorer/src/lib/components/block-details/BlockDetails.svelte

```diff
@@ -172,5 +188,30 @@
         )}</span
       >
     </ListItem>
+
+    <!-- PAYLOAD -->
+    <ListItem tooltipText="The payload">
+      <svelte:fragment slot="term">
+        payload
+
+        <Switch
+          className="block-details__payload-switch"
+          onSurface={true}
+          bind:value={isPayloadToggled}
+        />
+      </svelte:fragment>
+
+      <svelte:fragment slot="definition">
+        {#if isPayloadToggled}
+          <Card onSurface={true} className="block-details__payload">
+            <pre>
```

Comment:
It should be:
```svelte
<DataGuard data={payload}>
  <pre>{payload
      ? JSON.stringify(JSON.parse(payload), null, 2)
      : null}</pre>
</DataGuard>
```

Otherwise you're inserting extra whitespaces and blank lines before and after the JSON.